spring-cloud-dataflow-server/pom.xml

```diff
@@ -168,6 +168,9 @@
 						<volumes>
 						  <volume>/tmp</volume>
 						</volumes>
+            <runCmds>
+              <run>wget -q https://repo.spring.io/snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/${project.version}/spring-cloud-dataflow-shell-${project.version}.jar -O /shell.jar</run>
```

Comment:
Sure, makes sense. Just so I understand the concept: is the idea to avoid downloading and use `assembly.xml` to copy the artifact produced by the spring-cloud-dataflow-parent's multimodule build? Trying to keep an eye out for [recursive dependency problems during the build](http://maven.apache.org/plugins/maven-assembly-plugin/faq.html#module-binaries).

Reply:
the main concept is to avoid using an external tool that may or may not be on a machine doing a build (or maybe lacks public internet for example), possibly resulting in a build failure. in the end, if the artifact is on disk already, might as well use that. this does imply both server and shell modules are built, but in a release situation that would be done, in dev, one would likely know which modules need to be rebuilt etc.

Reply:
@chrisjs OK, may have to reorder the shell and server module builds. Let me see if I can get this to work with assembly.